### PR TITLE
chore: remove admin events feature flag

### PR DESCRIPTION
**Pull request by stehessel**
### Description

Remove the admin events feature flag and turn it on permanently.

### User-facing documentation

(*must be* 2 items and both *must be* checked)

- [x] CHANGELOG is updated
- [ ] CHANGELOG update is not needed
- [ ] Documentation PR is created and linked above
- [x] Documentation is not needed

### Testing

- [x] inspected CI results

#### Automated testing

(*must be* at least 1 item and all items *must be* checked)

- [ ] added unit tests
- [ ] added e2e tests
- [ ] added regression tests
- [ ] added compatibility tests
- [ ] modified existing tests
- [x] contributed **no automated tests**
  

#### How I validated my change

change me!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
 - ROX-23155: EBPF collection has been removed. If EBPF is configured, it
   will be automatically converted to CORE_BPF. forceCollection no longer has an
   effect.
+- The `ROX_ADMINISTRATION_EVENTS` feature flag has been removed. It's effect is now enabled by default.
```
Comment (0x656b694d) [inline CHANGELOG.md:49]:
```suggestion
- The `ROX_ADMINISTRATION_EVENTS` feature flag has been removed. Its effect is now enabled by default.
```

File: central/config/datastore/singleton.go [hunk 2]
```diff
@@ -128,30 +125,24 @@ func initialize() {
...
-		needsUpsert = true
 	}
 
-	if needsUpsert {
-		utils.Must(d.UpsertConfig(ctx, &storage.Config{
-			PublicConfig:  config.GetPublicConfig(),
-			PrivateConfig: privateConfig,
-		}))
-	}
+	utils.Must(d.UpsertConfig(ctx, &storage.Config{
```
Comment (0x656b694d) [inline central/config/datastore/singleton.go:142]: Why this change?

Comment (stehessel) [reply central/config/datastore/singleton.go:142]: With removing the feature flag, `needsUpsert` became always true. So I removed it.

**Review by dhaus67 — APPROVED** [review]
